scss/_patterns_form-tick-elements.scss

```diff
@@ -1,6 +1,13 @@
 @import 'settings';
 
 @mixin vf-p-form-tick-elements {
+  .p-checkbox,
+  .p-radio {
+    // Fix label alignment when text drops to new line
+    margin-left: $sph-inner + $form-tick-box-size;
```

Comment:
It seems that using margin breaks the `u-truncate` utility:

https://vanilla-framework-3965.demos.haus/docs/examples/utilities/truncate
<img width="1205" alt="Screenshot 2021-09-07 at 11 43 37" src="https://user-images.githubusercontent.com/83575/132324603-dd5b67f4-8f5f-46df-a27e-a2cd8e38d2a7.png">

I think using padding instead should work better.

```suggestion
    padding-left: $sph-inner + $form-tick-box-size;
```